pokebattle/settings/local_base.py

```diff
@@ -26,7 +26,7 @@
 AUTH_PASSWORD_VALIDATORS = []  # allow easy passwords only on local
 
 # Celery
-CELERY_TASK_ALWAYS_EAGER = True
+CELERY_TASK_ALWAYS_EAGER = False
```

Comment:
Not a problem, but this can be defined inside an ignored file by the source control management tool(i.e. git), so that if others clone the repo they wouldn't be required to set up RabbitMQ.